automation/nightly/test-nightly-build.sh

```diff
@@ -37,25 +37,45 @@ latest_cdi_image=$(curl -sL "https://storage.googleapis.com/kubevirt-prow/devel/
 IFS=: read -r cdi_image cdi_tag <<< "${latest_cdi_image}"
 latest_cdi_commit=$(curl -sL "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/containerized-data-importer/${latest_cdi}/commit")
 
+latest_cnao=$(curl -sL https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/cluster-network-addons-operator/latest)
+latest_cnao_image=$(curl -sL "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/cluster-network-addons-operator/${latest_cnao}/operator.yaml" | grep "image:" | grep "cluster-network-addons-operator" | sed -E "s|^ +image: (.*)$|\1|")
```

Comment:
nit:
i think it would be more "clean" to extract the CNAO image by using the yaml structure, instead of trying to catch it with grep and sed:
```suggestion
latest_cnao_image=$(curl -sL "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/cluster-network-addons-operator/${latest_cnao}/operator.yaml" | yq -r 'select(.kind=="Deployment").spec.template.spec.containers[0].image')
```
however, for this we need `yq`, which shouldn't be difficult to get from https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64, IMHO

Reply:
This is a very good suggestion, and it much simpler. However, yq is not installed by default in the automation and I don't want to maintain its installation just for this line.

Reply:
ok.
please note we can also use it for lines 30 and 36 above, that's 3 usages

Reply:
you can use the dockerized: https://hub.docker.com/r/linuxserver/yq
note I didn't try this specific image, but it looks promising